### PR TITLE
Rbac for workflows initial integration

### PR DESCRIPTION
**Pull request by marosmars**
End-to-end integration works ... between workflow UI and grapQL API providing user groups.

### CODE REVIEW

**Review by KenG98 — CHANGES_REQUESTED** [review]
This all looks good to me except one thing - it breaks eslint. Sorry eslint doesn't run in our CI. I've filed a task, I hope shortly it will (they recently started running jest tests and flow checks in CI, so eslint is close behind)
Here's how to reproduce:
```
 yarn eslint ./fbcnms-projects/hub/app/components/Main.js
```

**Comment by hotlib** [comment]
> This all looks good to me except one thing - it breaks eslint. Sorry eslint doesn't run in our CI. I've filed a task, I hope shortly it will (they recently started running jest tests and flow checks in CI, so eslint is close behind)
> Here's how to reproduce:
> 
> ```
>  yarn eslint ./fbcnms-projects/hub/app/components/Main.js
> ```

should be fixed, pls check :)

File: symphony/app/fbcnms-projects/hub/app/components/Main.js [hunk 1]
```diff
@@ -16,15 +16,18 @@ import Button from '@fbcnms/ui/components/design-system/Button';
...
 import Shuffle from '@material-ui/icons/Shuffle';
 import Text from '@fbcnms/ui/components/design-system/Text';
 import TextInput from '@fbcnms/ui/components/design-system/Input/TextInput';
 import WorkflowApp from './workflow/App';
+import WorkflowServiceApp from './workflow/ServiceUiApp';
 import nullthrows from '@fbcnms/util/nullthrows';
 import {Redirect, Route, Switch} from 'react-router-dom';
+import {conductorApiUrlPrefix} from './workflow/constants';
 import {getProjectLinks} from '@fbcnms/projects/projects';
+import {HttpClient as http} from './workflow/common/HttpClient';
```
Comment (tomasol) [inline symphony/app/fbcnms-projects/hub/app/components/Main.js:30]: I would like to suggest deprecating or at least limiting use of this module as it has some questionable design decisions like calling both resolve and reject, calling resolve with response body or status code in the same method etc.

Comment (KenG98) [reply symphony/app/fbcnms-projects/hub/app/components/Main.js:30]: I've already merged the PR, but thanks for the good feedback. Feel free to open another PR if necessary.

**Pull request closed without merging**